tasks/setup-RedHat.yml

```diff
@@ -1,21 +1,13 @@
 ---
-- name: Enable GlusterFS yum repository.
-  copy:
-    src: glusterfs-epel.repo
-    dest: /etc/yum.repos.d/glusterfs-epel.repo
-    owner: root
-    group: root
-    mode: 0644
-
-- name: Import GlusterFS GPG key.
-  rpm_key:
-    key: "http://download.gluster.org/pub/gluster/glusterfs/LATEST/rsa.pub"
-    state: present
+- name: Install Prerequisites
+  yum: name={{ item }}  state=present
+  with_items:
+    - "centos-release-gluster{{ glusterfs_default_release }}"
 
-- name: Ensure GlusterFS is installed.
-  yum:
-    name: "{{ item }}"
-    state: installed
+- name: Install Packages
+  yum: name={{ item }}  state=present
   with_items:
     - glusterfs-server
     - glusterfs-client
+
+...
```

Comment:
While true... that's one convention that I've rarely seen used in practice. I had forgotten that it's valid syntax (been a year or two since I last read through the reference :D).

Since it's not used elsewhere, can you remove it here?